### PR TITLE
[iOS] update color scheme for dark mode

### PR DESCRIPTION
**Pull request by tkhs0604**
## Issue
- close #ISSUE_NUMBER

## Overview (Required)
- Update color scheme for dark mode
- Regarding custom colors, I am updating them in https://github.com/DroidKaigi/conference-app-2023/pull/724 for light mode. So let me update them after both PRs being merged 🙏

## Links
- 

## Screenshot
Light | Dark
:--: | :--:
 |

### CODE REVIEW

File: app-ios/Modules/Sources/Theme/Resources/Colors.xcassets/Outline.colorset/Contents.json
Comment (tkhs0604) [inline app-ios/Modules/Sources/Theme/Resources/Colors.xcassets/Outline.colorset/Contents.json:1]: Move it under Outline package

**Comment by tkhs0604** [comment]
Looks like `foregroundStyle` on the bookmark icons don't work as expected originally?

**Comment by tkhs0604** [comment]
> Seems like `foregroundStyle` on the bookmark icons don't work as expected originally?

Ah, I found the cause.
We need to change `Render As` option from "Default" to "Template Image" 👀
https://tech.nri-net.com/entry/swift_renderingmode_colored_image

Let me fix in another PR.

**Review by ry-itto — APPROVED** [review]
So cool! Thank you!! :pray:

**Pull request merged**